docs/apache-airflow/core-concepts/objectstorage.rst

```diff
@@ -305,6 +311,27 @@ Airflow needs to walk the directory tree and copy each file individually. This i
 source to the target.
 
 
+Caching
```

Comment:
Should we separate it out ? I think that's unrelated to open lineage?

Reply:
It is indeed unrelated and I am piggy backing here :-). Objections to keeping it? Otherwise I will separate it.

Reply:
I think it's fine to keep several unrelated changes in "suppplementary" code - like dev tools etc - we are always cherry-picking pretty much "all" changes there. However in the "core" code it's very important to keep unrelated changes - even documentation ones - separate - because it makes such changes easier cherry-pickable and makes release notes cleaner and easier to follow (those two changes should be mentioned in different sections of the relase notes).

Simply speaking - by having unrelated changes in one PR you make your life slightly easier while release manager life quite a lot more complex.

So yes, I think it should be split.

Reply:
Okido. I'll split it.